dream_layer_backend/shared_utils.py

```
@@ -228,41 +218,62 @@ def serve_image(filename: str) -> Any:
     This is a shared function used by all servers
     """
     from flask import send_file, jsonify
```

Ruff (0.12.2)

Redefinition of unused `send_file` from line 9

Remove definition: `send_file`

(F811)